### PR TITLE
Made doc translations in Hindi and Chinese

### PR DESCRIPTION
**Pull request by virejdasani**
Added a new Chinese translated document in a new folder called "ch-CH" inside the docs-translations folder.
This documents has been proofread.

### CODE REVIEW

**Review by virejdasani — COMMENTED** [review]
Added a Hindi translated document in the docs-translations folder in the hin-HIN folder

**Comment by kevin-bates** [comment]
Hi @virejdasani - thank you for this contribution!  I am unable to review these changes as I don't know how to read Chinese or Hindi but there are some changes that will be necessary prior to its merge.

- The language and country codes are not correct.  These correspond to the 'ch-CH' and 'hin-HIN' directory names that contain the translated files.  I believe we're trying to follow the [ISO 639 (language codes)](http://www.loc.gov/standards/iso639-2/php/code_list.php) and [ISO 3166 (country codes)](http://kirste.userpage.fu-berlin.de/diverse/doc/ISO_3166.html) standards for specifying these, which, together, essentially comprise the user's _locale_.

- For Chinese translations, the language code is 'zh' and you'll typically see a country code of 'CN' (for mainland China) or 'TW' (for Taiwan) (there are also 'HK' - Hong Kong and 'SG' - Singapore, etc.)  The 'CH' country code actually corresponds to Switzerland.  

    Please relocate your Chinese-translated files to a 'zh-CN' directory (or one of the other country codes if more appropriate).

- While 'hin' is a valid language code for Hindi, I think we should stick with the two-character codes and use 'hi'.  Since Hindi is not a country, we need to identify a particular region/country to which we can map your Hindi translations.  Looking at a list of Hindi-speaking countries, I suspect that the India country code 'IN' would be the appropriate indicator.  Nepal's country code (which has the second-largest number of Hindi-speaking people) is 'NP' (in case that's more appropriate). 

    Please relocate your Hindi-translated files to a 'hi-IN' directory (or use a different country code as appropriate).

- Also, please update the title to indicate that Hindi translations are also included.

**Comment by virejdasani** [comment]
Thank you for specifying this and i'm sorry I was not aware of this.
I have gone ahead and made these changes but I am not able to create a new Pull Request.
As this is my first time, i'm not too sure weather I have to make a new pull request or if it happens automatically.
Either way, if you need me to do anything else, just mention it.
Thank You.

**Comment by kevin-bates** [comment]
What you've done Virej is perfect.  You can keep pushing commits to your branch and it will update the changes for this pull request.  Only if you needed to make changes for a different PR would you want to move to a different branch.  You should be able to edit the title using the 'Edit' button near the title itself.

As I noted earlier, I'm not qualified to review the content-related changes that you've made.  I'm hoping others from the community can take a few minutes and provide some reviews.  As a result, I'd prefer to keep this open over the weekend, and perhaps by Tuesday of next week, assuming no changes have been requested, I'll go ahead with the merge.

Thank you again!

**Comment by virejdasani** [comment]
Sorry for this,
I just found out that I have to make a new PR for my contributions to be merged.
Since I already have one PR on this branch, apparently I have to switch to another branch to make the PR.
Is this what i'm supposed to do, and if so, which branch should I make the PR from?

**Comment by virejdasani** [comment]
Done.
Thanks a lot!

**Comment by virejdasani** [comment]
Sorry to bother you but, I'd appreciate it if you could add the hacktoberfest-approved tag when merging.
Thanks in advance!

**Comment by kevin-bates** [comment]
> I'd appreciate it if you could add the hacktoberfest-approved tag when merging.

hi @virejdasani - I'm new to maintaining this repository (and to GH in general) and am not aware of maintainers making any alterations during merge operations (other than perhaps removing some comments when squashing commits).

I'm assuming this is in reference to Digital Ocean's Open Source Hacktoberfest and the tag you're referring to is #Hacktoberfest.  Looking at the [FAQ](https://hacktoberfest.digitalocean.com/faq), this doesn't appear necessary for you to receive credit:
>Do issues have to be tagged #Hacktoberfest to count?
No, any pull request made on a public repository counts, whether it’s attached to a hacktoberfest issue or not.

so I'm inclined to continue performing the usual merge operations.  We really appreciate your contribution and hope you continue to improve the Jupyter ecosystem.  

If other maintainers know of what steps to perform to accommodate things like Hactoberfest, please comment.  Thanks.

**Comment by virejdasani** [comment]
You are correct, I'm sorry, there was a misunderstanding on my part

File: docs-translations/hi-IN/README.md
Comment (kevin-bates) [inline docs-translations/hi-IN/README.md:12]: The [Notice block](https://github.com/jupyter/notebook#notice) appears to be missing.

File: docs-translations/zh-CN/README.md
Comment (kevin-bates) [inline docs-translations/zh-CN/README.md:12]: same as previous regarding the Notice block.

**Review by kevin-bates — COMMENTED** [review]
Thanks Virej.  Please add the Notice block so we can communicate the current state of affairs to our Hindi and Chinese users.  Could you also make a pass to ensure the rest of the content reflects the current README file?  Thank you.

File: docs-translations/zh-CN/README.md
Comment (virejdasani) [reply docs-translations/zh-CN/README.md:12]: I'm sorry, i'm not entirely sure what I have to do.
I'm new to this sort of thing and I couldn't find what Notice block meant.
Again, I apologize but could you briefly explain what exactly I have to do

Comment (kevin-bates) [reply docs-translations/zh-CN/README.md:12]: No worries.  
These translations are translations of the README.md file.  We've recently added a section in the README.md labelled "Notice", just following the Notebook image and preceding the heading "Jupyter notebook, the language-agnostic evolution of IPython notebook".  Your translations don't appear to contain that block of text and I think it would be a good idea to include that information since it's important to users and contributors.  I included a link to this section of the README in the previous comment above, and [here as well](https://github.com/jupyter/notebook#notice).
Thank you.

Comment (virejdasani) [reply docs-translations/zh-CN/README.md:12]: Sorry for all this time, I had my exams in school

Comment (kevin-bates) [reply docs-translations/zh-CN/README.md:12]: No worries - I hope your exams went well.  Will you be updating the Chinese translation as well?  No rush.

Comment (virejdasani) [reply docs-translations/zh-CN/README.md:11]: I have updated both the documents

**Review by kevin-bates — APPROVED** [review]
Thank you Virej.

**Comment by virejdasani** [comment]
Will you be merging them?

**Comment by virejdasani** [comment]
Thanks a lot!

**Pull request merged**
